models/issues/issue_project.go

```diff
@@ -51,7 +51,7 @@ func (issue *Issue) ProjectBoardID(ctx context.Context) int64 {
 func LoadIssuesFromBoard(ctx context.Context, b *project_model.Board) (IssueList, error) {
 	issueList := make(IssueList, 0, 10)
 
-	if b.ID != 0 {
+	if b.ID != 0 && !(b.ID == -1 && b.Default) {
```

Comment:
Use constant variable but -1 directly.

Reply:
I don't understand what you mean.  Can you please clarify?

Reply:
What lunny means is you should use a variable instead of the constant.
I'd even go a step further, and the whole check should be extracted into its own function.

Reply:
The code used -1 since the Kanban board was introduced in 2020. Maybe should do some refactor later.

Reply:
I can simplify the check to `b.ID > 0` to cover all cases.